src/XR.tsx

```diff
@@ -17,6 +17,7 @@ export interface XRState {
   player: THREE.Group
   session: XRSession | null
   foveation: number
+  frameRate?: number
```

Comment:
We could write `XRSession.frameRate` here rather than having this `number | undefined` so people could procedurally clamp this down. Could this be undefined or not present on the XRSession though?

Reply:
Yes, most of the times it is actually undefined, so I don't think it makes sense